source/_components/binary_sensor.lupusec.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: "Lupusec Binary Sensor"
+description: "Instructions on how to integrate Lupusec binary sensors into Home Assistant."
+date: 2018-10-03 16:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lupusec.jpg
+ha_category: Binary Sensor
+ha_release: 0.81
+ha_iot_class: "Local Push"
+---
+
+The `LUPUS-Electronics` security control panel platform allows you to control your [Lupusec](https://www.lupus-electronics.de) devices.
+
+This component will add `door contacts` and `window contacts` to your home-assistant setup
+
+For the switches to work you have to setup your [Lupusec component](/components/lupusec/).
```

Comment:
It's confusing to call them switches. Switches in home assistant are actuator devices. The page is for binary sensors.